### PR TITLE
Add live test for Oghliner used as a template with CLI commands

### PR DESCRIPTION
**Pull request by marco-c**

### CODE REVIEW

**Comment by marco-c** [comment]
I need to update this to support 2FA like #105.

File: test/liveUtils.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+      });
+
+      resolve(res);
+    });
+  }).catch(function(err) {
+    var error = JSON.parse(err.message);
+
+    if (error.message === 'Must specify two-factor authentication OTP code.') {
+      useOTP = true;
+      return createAuthorization(uesrname, password);
```
Comment (mykmelez) [inline test/liveUtils.js]: uesrname -> username

Comment (marco-c) [inline test/liveUtils.js]: Right, you made me remember that I haven't tested it yet with 2FA:

File: test/liveUtils.js [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+var github = new GitHub({
+  version: '3.0.0',
+  protocol: 'https',
+  headers: {
+    'user-agent': 'Oghliner',
+  },
+});
+
+var liveUtils = {
+  repoName: 'test_oghliner_live' + process.version + process.pid,
```
Comment (mykmelez) [inline test/liveUtils.js]: Nit: these repo names would be a bit easier to read with separator characters between the version and id, f.e.: `test_oghliner_livev0.12.729326` -> `test_oghliner_live_v0.12_729326`.

**Comment by mykmelez** [comment]
It would be helpful to make test cleanup optional, so you can test _configure_ with both a new repo and an existing one. That'd also require us to reuse repo names instead of generating a new one each time. We can do this in a followup

**Comment by mykmelez** [comment]
Looks like this needs conflict resolution since I merged a couple other fixes.

**Comment by marco-c** [comment]
> It would be helpful to make test cleanup optional, so you can test configure with both a new repo and an existing one. That'd also require us to reuse repo names instead of generating a new one each time. We can do this in a followup

Filed #164.

**Comment by marco-c** [comment]
Merging this so I can update #159.

**Pull request merged**